www/redland-bindings/Portfile

```diff
@@ -35,33 +32,40 @@ pre-configure {
     }
 }
 
-variant perl5 {
+default_variants +python37
+
+variant perl5 description {Include Perl 5 binding} {
 	depends_lib-append path:bin/perl:perl5
```

Comment:
make sure here (and for other variants) that no tabs are used; please replace with four spaces